frontend/public/components/_terminal.scss

```diff
@@ -11,4 +11,14 @@
   height: 100%;
   padding: 0;
   background-color: $color-pf-black;
+  position: relative;
+  z-index: 1040; // to get above 'terminal xterm fullscreen's z-index of 255 and $zindex-navbar-fixed of 1030
+}
+
+.console-collapse-link {
+  color: $color-pf-blue-300;
+  position: fixed;
+  right: 18px;
+  top: 4px;
+  z-index: 1045;  // to get console's z-index of 1040
```

Comment:
tempted to ask you to make variables so they can be declared side-by-side and the relationship is clear. (if you ever bump console z-index, this needs to change too. would be nice to have them declared together)

Reply:
Then you could even use math like console-collapse-link-z-index = console-z-index + 1

Reply:
Should those new vars go in `console/frontend/public/style/_vars.scss` or `components/_terminal.scss`?

Reply:
Put vars into `_terminal.scss`, let me know if that is incorrect -thanks